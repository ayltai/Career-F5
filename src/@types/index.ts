export type { Course, } from './Course';
export type { Experience, } from './Experience';
export type { Profession, } from './Profession';
export type { RouteItem, } from './RouteItem';
export type { Section, } from './Section';
export type { Segmentation, } from './Segmentation';
