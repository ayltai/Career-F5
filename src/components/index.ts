export { ApplyAction, } from './ApplyAction';
export { ApplyDialog, } from './ApplyDialog';
export { ContactAction, } from './ContactAction';
export { Footer, } from './Footer';
export { PaymentPlan, } from './PaymentPlan';
export { RangeIndicator, } from './RangeIndicator';
export { TopBar, } from './TopBar';
