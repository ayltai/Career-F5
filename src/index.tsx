import * as Sentry from '@sentry/react';
import mixpanel from 'mixpanel-browser';
import React, { StrictMode, } from 'react';
import ReactDOM from 'react-dom/client';
import { Provider, } from 'react-redux';
import { PersistGate, } from 'redux-persist/integration/react';
import '@fontsource/kreon/400.css';
import '@fontsource/kreon/700.css';
import '@fontsource/lobster-two/400.css';
import '@fontsource/noto-sans-tc/chinese-traditional-400.css';
import '@fontsource/noto-sans-tc/chinese-traditional-700.css';
import '@fontsource/noto-serif-tc/chinese-traditional-400.css';
import '@fontsource/noto-serif-tc/chinese-traditional-700.css';
import '@fontsource/sansita/400.css';
import '@fontsource/sansita/400-italic.css';
import '@fontsource/sansita/700.css';

import PackageInfo from '../package.json';

import { App, } from './App';
import { applyI18n, } from './i18n';
import en from './i18n/en.json';
import zh from './i18n/zh.json';
import { persistor, store, } from './states';
import { handleError, } from './utils';
import './index.css';

if (process.env.REACT_APP_MIXPANEL_TOKEN) mixpanel.init(process.env.REACT_APP_MIXPANEL_TOKEN, {
    debug : process.env.NODE_ENV !== 'production',
});

if (process.env.REACT_APP_SENTRY_DSN) Sentry.init({
    dsn              : process.env.REACT_APP_SENTRY_DSN,
    environment      : process.env.NODE_ENV || 'development',
    release          : PackageInfo.version,
    tracesSampleRate : 1,
    integrations     : [
        new Sentry.BrowserTracing(),
    ],
});

applyI18n({
    language         : navigator.language.substring(0, 2),
    fallbackLanguage : 'en',
    resources        : {
        en : {
            translation : en,
        },
        zh : {
            translation : zh,
        },
    },
}).then(() => ReactDOM.createRoot(document.getElementById('root')!).render(
    <StrictMode>
        <Sentry.ErrorBoundary showDialog>
            <Provider store={store}>
                <PersistGate persistor={persistor}>
                    <App />
                </PersistGate>
            </Provider>
        </Sentry.ErrorBoundary>
    </StrictMode>
)).catch(handleError);
