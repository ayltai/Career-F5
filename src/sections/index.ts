export { BannerSection, } from './BannerSection';
export { ContactSection, } from './ContactSection';
export { CourseSection, } from './CourseSection';
export { InstructorSection, } from './InstructorSection';
export { OverviewSection, } from './OverviewSection';
export { PaymentSection, } from './PaymentSection';
export { ProspectSection, } from './ProspectSection';
export { StructureSection, } from './StructureSection';
