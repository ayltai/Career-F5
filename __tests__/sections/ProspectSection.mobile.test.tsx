import React from 'react';

import { segmentations, } from '../../src/i18n/en.json';
import { createMatchMedia, render, } from '../../src/utils/test';
import { ProspectSection, } from '../../src/sections';

describe('ProspectSection', () => {
    beforeAll(() => {
        window.matchMedia = createMatchMedia(360);
    });

    it('renders correctly for mobile', () => expect(render(
        <ProspectSection
            category='category'
            title='title'
            description='description'
            segmentations={segmentations} />
    )).toMatchSnapshot());
});
