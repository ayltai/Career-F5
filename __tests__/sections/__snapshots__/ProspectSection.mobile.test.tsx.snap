// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProspectSection renders correctly for mobile 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-1jo1901"
      >
        <div
          class="MuiStack-root css-153idv-MuiStack-root"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-1ceq6vm-MuiTypography-root"
          >
            category
          </h6>
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-8ts8qi-MuiTypography-root"
          >
            title
          </h3>
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-tvi8ch-MuiTypography-root"
          >
            description
          </h6>
          <div
            class="MuiBox-root css-19midj6"
          >
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary  css-zg7km9-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
            >
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
                role="button"
                tabindex="0"
              >
                location_hongkong
              </div>
              <input
                aria-hidden="true"
                aria-invalid="false"
                class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                tabindex="-1"
                value="location_hongkong"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-ihdtdm"
                >
                  <span
                    class="notranslate"
                  >
                    ​
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiBox-root css-19midj6"
          >
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary  css-zg7km9-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
            >
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
                role="button"
                tabindex="0"
              >
                prospect_per_month
              </div>
              <input
                aria-hidden="true"
                aria-invalid="false"
                class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                tabindex="-1"
                value="month"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-ihdtdm"
                >
                  <span
                    class="notranslate"
                  >
                    ​
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiBox-root css-ph8oxj"
          >
            <div
              class="MuiStack-root css-8cva6s-MuiStack-root"
            >
              <div
                class="MuiStack-root css-1t2ey5u-MuiStack-root"
              >
                <div
                  class="MuiBox-root css-i9gxme"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    Backend Software Engineer
                  </p>
                </div>
                <div
                  class="MuiBox-root css-2uchni"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    HK$51,000
                  </p>
                </div>
              </div>
              <span
                class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
              >
                <span
                  class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
                />
                <span
                  class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                  style="left: 12.885000000000002%; width: 68.265%;"
                />
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="0"
                  style="left: 12.885000000000002%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="21475"
                    data-index="0"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="21475"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$21,475
                      </span>
                    </span>
                  </span>
                </span>
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="1"
                  style="left: 81.15%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="135250"
                    data-index="1"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="135250"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$135,250
                      </span>
                    </span>
                  </span>
                </span>
              </span>
            </div>
            <div
              class="MuiStack-root css-8cva6s-MuiStack-root"
            >
              <div
                class="MuiStack-root css-1t2ey5u-MuiStack-root"
              >
                <div
                  class="MuiBox-root css-i9gxme"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    Web Engineer
                  </p>
                </div>
                <div
                  class="MuiBox-root css-2uchni"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    HK$81,375
                  </p>
                </div>
              </div>
              <span
                class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
              >
                <span
                  class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
                />
                <span
                  class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                  style="left: 17.500000000000004%; width: 44.19999999999999%;"
                />
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="0"
                  style="left: 17.500000000000004%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="29166.666666666668"
                    data-index="0"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="29166.666666666668"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$29,167
                      </span>
                    </span>
                  </span>
                </span>
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="1"
                  style="left: 61.699999999999996%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="102833.33333333333"
                    data-index="1"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="102833.33333333333"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$102,833
                      </span>
                    </span>
                  </span>
                </span>
              </span>
            </div>
            <div
              class="MuiStack-root css-8cva6s-MuiStack-root"
            >
              <div
                class="MuiStack-root css-1t2ey5u-MuiStack-root"
              >
                <div
                  class="MuiBox-root css-i9gxme"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    DevOps Engineer
                  </p>
                </div>
                <div
                  class="MuiBox-root css-2uchni"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                  >
                    HK$66,500
                  </p>
                </div>
              </div>
              <span
                class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
              >
                <span
                  class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
                />
                <span
                  class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                  style="left: 13.000000000000002%; width: 59.900000000000006%;"
                />
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="0"
                  style="left: 13.000000000000002%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="21666.666666666668"
                    data-index="0"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="21666.666666666668"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$21,667
                      </span>
                    </span>
                  </span>
                </span>
                <span
                  class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                  data-index="1"
                  style="left: 72.9%;"
                >
                  <input
                    aria-orientation="horizontal"
                    aria-valuemax="166666.66666666666"
                    aria-valuemin="0"
                    aria-valuenow="121500"
                    data-index="1"
                    max="166666.66666666666"
                    min="0"
                    step="1"
                    style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                    type="range"
                    value="121500"
                  />
                  <span
                    aria-hidden="true"
                    class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                  >
                    <span
                      class="MuiSlider-valueLabelCircle"
                    >
                      <span
                        class="MuiSlider-valueLabelLabel"
                      >
                        HK$121,500
                      </span>
                    </span>
                  </span>
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-1jo1901"
    >
      <div
        class="MuiStack-root css-153idv-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-1ceq6vm-MuiTypography-root"
        >
          category
        </h6>
        <h3
          class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-8ts8qi-MuiTypography-root"
        >
          title
        </h3>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-tvi8ch-MuiTypography-root"
        >
          description
        </h6>
        <div
          class="MuiBox-root css-19midj6"
        >
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary  css-zg7km9-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
              role="button"
              tabindex="0"
            >
              location_hongkong
            </div>
            <input
              aria-hidden="true"
              aria-invalid="false"
              class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
              tabindex="-1"
              value="location_hongkong"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-ihdtdm"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiBox-root css-19midj6"
        >
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary  css-zg7km9-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
              role="button"
              tabindex="0"
            >
              prospect_per_month
            </div>
            <input
              aria-hidden="true"
              aria-invalid="false"
              class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
              tabindex="-1"
              value="month"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-ihdtdm"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiBox-root css-ph8oxj"
        >
          <div
            class="MuiStack-root css-8cva6s-MuiStack-root"
          >
            <div
              class="MuiStack-root css-1t2ey5u-MuiStack-root"
            >
              <div
                class="MuiBox-root css-i9gxme"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  Backend Software Engineer
                </p>
              </div>
              <div
                class="MuiBox-root css-2uchni"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  HK$51,000
                </p>
              </div>
            </div>
            <span
              class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
            >
              <span
                class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
              />
              <span
                class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                style="left: 12.885000000000002%; width: 68.265%;"
              />
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="0"
                style="left: 12.885000000000002%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="21475"
                  data-index="0"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="21475"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$21,475
                    </span>
                  </span>
                </span>
              </span>
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="1"
                style="left: 81.15%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="135250"
                  data-index="1"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="135250"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$135,250
                    </span>
                  </span>
                </span>
              </span>
            </span>
          </div>
          <div
            class="MuiStack-root css-8cva6s-MuiStack-root"
          >
            <div
              class="MuiStack-root css-1t2ey5u-MuiStack-root"
            >
              <div
                class="MuiBox-root css-i9gxme"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  Web Engineer
                </p>
              </div>
              <div
                class="MuiBox-root css-2uchni"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  HK$81,375
                </p>
              </div>
            </div>
            <span
              class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
            >
              <span
                class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
              />
              <span
                class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                style="left: 17.500000000000004%; width: 44.19999999999999%;"
              />
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="0"
                style="left: 17.500000000000004%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="29166.666666666668"
                  data-index="0"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="29166.666666666668"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$29,167
                    </span>
                  </span>
                </span>
              </span>
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="1"
                style="left: 61.699999999999996%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="102833.33333333333"
                  data-index="1"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="102833.33333333333"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$102,833
                    </span>
                  </span>
                </span>
              </span>
            </span>
          </div>
          <div
            class="MuiStack-root css-8cva6s-MuiStack-root"
          >
            <div
              class="MuiStack-root css-1t2ey5u-MuiStack-root"
            >
              <div
                class="MuiBox-root css-i9gxme"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  DevOps Engineer
                </p>
              </div>
              <div
                class="MuiBox-root css-2uchni"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1ml4jyn-MuiTypography-root"
                >
                  HK$66,500
                </p>
              </div>
            </div>
            <span
              class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium css-1i3537d-MuiSlider-root"
            >
              <span
                class="MuiSlider-rail css-14pt78w-MuiSlider-rail"
              />
              <span
                class="MuiSlider-track css-1gv0vcd-MuiSlider-track"
                style="left: 13.000000000000002%; width: 59.900000000000006%;"
              />
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="0"
                style="left: 13.000000000000002%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="21666.666666666668"
                  data-index="0"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="21666.666666666668"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$21,667
                    </span>
                  </span>
                </span>
              </span>
              <span
                class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary css-3nibdl-MuiSlider-thumb"
                data-index="1"
                style="left: 72.9%;"
              >
                <input
                  aria-orientation="horizontal"
                  aria-valuemax="166666.66666666666"
                  aria-valuemin="0"
                  aria-valuenow="121500"
                  data-index="1"
                  max="166666.66666666666"
                  min="0"
                  step="1"
                  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
                  type="range"
                  value="121500"
                />
                <span
                  aria-hidden="true"
                  class="MuiSlider-valueLabel css-1jknfwc-MuiSlider-valueLabel"
                >
                  <span
                    class="MuiSlider-valueLabelCircle"
                  >
                    <span
                      class="MuiSlider-valueLabelLabel"
                    >
                      HK$121,500
                    </span>
                  </span>
                </span>
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
